Makefile

```
@@ -1,5 +1,6 @@
 BRANCH=`git rev-parse --abbrev-ref HEAD`
 COMMIT=`git rev-parse --short HEAD`
+LATEST_RELEASE_BRANCH := release-$(shell grep -ohE "[0-9]+.[0-9]+" version/version.go)
```

The issue with the current workflow failure is that it's trying to fetch the `release-1.4` branch. However, because the `main` branch has the next release, this should return `1.3` rather than `1.4`.

```suggestion
LATEST_RELEASE_BRANCH := release-$(shell grep -ohE "[0-9]+.[0-9]+" version/version.go | awk -F. '{print $1 "." $2-1})'
```